include/light/light.hh

```diff
@@ -171,7 +171,9 @@ enum class lightfile
     external = 1,
     bspx = 2,
     both = external | bspx,
-    lit2 = 4
+    lit2 = 4,
+    hdr = 8,
+    bspxhdr = 16,
```

Comment:
Added a separate enum-entry here to be able to distinguish between rgb lit and e5bgr9 lit. I'm not particularly happy with the relation between these different flags. Need to have bspxhdr here to be able to write only e5bgr9 data, otherwise both that and rgb data would be written. With this setup -bspxlit and -bspxhdr would write both.

Reply:
Same here, I'm not sure if the `enum class lightfile` really does anything useful, thinking of removing it (later on) and just checking flags like `light_options.lit.value()` directly.